src/shogun/mathematics/linalg/internal/implementation/MeanEigen3.h

```diff
@@ -73,11 +75,24 @@ struct int2float<int32_t>
 template <>
 struct int2float<int64_t>
 {
+	/** float64_t type **/
 	using floatType = float64_t;
 };
- 
+#else
```

Comment:
I am not sure I understand the logic of this preprocessor if then else.
Don't we just want to hide things from swig? Why the else case?